test/analytics/tests/instrumented_events/from_the_browser/core_context_providers.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { Event } from '@kbn/core/public';
+import { FtrProviderContext } from '../../../services';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const deployment = getService('deployment');
+  const ebtUIHelper = getService('kibana_ebt_ui');
+  const { common } = getPageObjects(['common']);
+
+  describe('Core Context Providers', () => {
+    let event: Event;
+    before(async () => {
+      await common.navigateToApp('home');
+      [event] = await ebtUIHelper.getLastEvents(1, ['Loaded Kibana']); // Get the loaded Kibana event
+    });
+
+    it('should have the properties provided by the "cluster info" context provider', () => {
+      expect(event.context).to.have.property('cluster_uuid');
+      expect(event.context.cluster_uuid).to.be.a('string');
+      expect(event.context).to.have.property('cluster_name');
+      expect(event.context.cluster_name).to.be.a('string');
+      expect(event.context).to.have.property('cluster_version');
+      expect(event.context.cluster_version).to.be.a('string');
+    });
+
+    it('should have the properties provided by the "build info" context provider', () => {
+      expect(event.context).to.have.property('isDev');
+      expect(event.context.isDev).to.be.a('boolean');
```

Comment:
I agree that it's redundant to the test. However, I kept it for now. I think it provides an additional level of detail in the tests (instead of saying `undefined` is not a `boolean`, it clearly states the missing property.

New tests can skip adding `to.have.property` if they want to :)